plugins/test/better-sort.js

```diff
@@ -0,0 +1,138 @@
+var assert = require('assert'),
```

Comment:
has this been rolled into test/TranslationServer.js?  seems redundant.  It can stay separate if that's better.